sal_interfaces/OCPS/OCPS_Events.xml

```diff
@@ -28,4 +28,31 @@
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>OCPS</Subsystem>
+    <EFDB_Topic>OCPS_logevent_job_id</EFDB_Topic>
+    <item>
+      <EFDB_Name>event_name</EFDB_Name>
```

Comment:
I think in order to maintain consistency in the attributes either write them as:
- eventName
- seqNum
- jobId

or
- event_name
- seq_num
- job_id

Reply:
Similarly, the field in `job_result` is `job_id`.  `seqNum` was for parallelism with its underlying source: `private_seqNum`, as that is an important detail needed for its usage.  But if we don't mind usage like `event.seq_num == trigger.private_seqNum`, I'm happy to change it.

Reply:
OK. If there is precedent for they way things are names, then they can all stay the way you originally wrote them.